.github/workflows/test.yml

```diff
@@ -18,6 +18,7 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: 2.7.1
+        bundler-cache: true
 
     - name: Install dependencies
```

Comment:
Think you can remove this call when the `bundler-cache` is on, since it also handles the install

Reply:
Oh, thanks for catching that!